plugin.xml

```diff
@@ -1,83 +1,72 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        id="cordova-admob-sdklibs"
-        version="2.1.4">
-      
+<plugin 
+    xmlns="http://apache.org/cordova/ns/plugins/1.0" 
+    xmlns:android="http://schemas.android.com/apk/res/android" id="cordova-admob-sdklibs" version="2.1.4">
     <name>Google Mobile Ads SDK for Cordova</name>
     <description>Add Google Mobile Ads SDK to Cordova project, as dependency for other plugins</description>
     <author>Google</author>
     <keywords>admob,google,ad</keywords>
     <repo>https://github.com/sunnycupertino/cordova-admob-sdklibs.gitt</repo>
     <issue>https://github.com/sunnycupertino/cordova-admob-sdklibs/issues</issue>
-
     <engines>
         <engine name="cordova" version=">=3.0" />
+        <engine name="cordova-android" version=">=6.3.0"/>
```

Comment:
@macdonst Is this a requirement for the change?

Reply:
@ratson yes, without it the environment variable won't be replaced correctly.